content/blog/2016/2016-09-22-jenkins-world-2016-wrap-up-pipeline.adoc

```diff
@@ -0,0 +1,141 @@
+implement other interesting scenarios such as early validation of pipelines and
+a visual pipeline editor.
+
+link:https://www.cloudbees.com/sites/default/files/2016-jenkins-world-introducing_a_new_way_to_define_jenkins_pipelines_1.pdf[*Download Presentation*]
+
+---
+
+image:/images/post-images/pipeline-at-jenkinsworld/perforce.png[Perforce, role=left]
+
+In
+link:https://www.cloudbees.com/birds-feather-session-perfecting-your-development-tools-updates-helix-plugin-jenkins[*Perfecting Your Development Tools: Updates to the Helix Plugin for Jenkins*],
+Paul Allen of Perforce walked through using Perforce's "Monorepo" model with Jenkins Pipeline.
+He explained in detail how to work with the Perforce "P4" plugin in Jenkins, including credential passing and workspace management.
+Of particular interest was his side-by-side comparison the various actions done with the Jenkins UI vs Pipeline.
+
+link:https://www.cloudbees.com/sites/default/files/2016-jenkins-world-an_introduction_to_the_p4_plugin_and_update_on_the_latest_features.pdf[*Download Presentation*]
+
+---
+
+image:/images/post-images/2016-jenkins-world-wrap/faster-pipelines-code.png[Building Pipelines To Be Faster,width=240,role=right]
+
+Sam Van Oort demonstrated strategies for faster pipelines in
+link:https://www.cloudbees.com/need-speed-building-pipelines-be-faster[*The Need For Speed: Building Pipelines To Be Faster*].
+He discussed various elements that contribute to making pipelines faster or slower,
+such a number of resources and latency.  He then showed several best practices
+for constructing pipelines that have lower turnaround times and reduced resource use.
+He also reviewed plugins and tools that can help analyze and visualize pipeline
+performance, including the Pipeline Stage View plugin and Blue Ocean.
+
+https://www.cloudbees.com/sites/default/files/2016-jenkins-world-the_need_for_speed-_building_pipelines_to_be_faster.pdf[*Download Presentation*]
+
+
+---
+
+image:/images/post-images/pipeline-at-jenkinsworld/redhat.png[role=right]
+
+link:https://www.cloudbees.com/how-do-continuous-delivery-jenkins-pipeline-docker-and-kubernetes[*How to Do Continuous Delivery with Jenkins Pipeline, Docker and Kubernetes*],
+presented by James Strachan of Red Hat, showed how to use Jenkins Pipeline with
+Docker and Kubernetes to implement a complete end-to-end continuous delivery and
+continuous improvement system using open source software for both microservices
+and monolithic applications. He demonstrated how to
+create or import projects, and have them automatically build, run
+system and integration tests, stage, and finally deploy. He also showed to
+manage and update those deployed applications using continuous
+delivery practices.
+
+---
+
+image:/images/post-images/scaling-jenkins-at-jenkinsworld/320px-CloudBees_official_logo.png[role=left]
+
+Bobby Sandell and James T. Nord talked about what not to do with Pipeline in
```

Comment:
In terms of name? I'm using the name as listed on https://www.cloudbees.com/no-you-shouldnt-do-lessons-using-pipeline